MapboxSearch/discover/src/main/java/com/mapbox/search/discover/DiscoverResult.kt

```diff
@@ -70,7 +82,9 @@ public class DiscoverResult internal constructor(
      * @suppress
      */
     override fun hashCode(): Int {
-        var result = name.hashCode()
+        var result = id.hashCode()
+        result = 31 * result + mapboxId.hashCode()
+        result = 31 * result + name.hashCode()
         result = 31 * result + address.hashCode()
```

Comment:
Same question here - does this function still make sense?

Reply:
Answered in my comment above